tests/test_core.py

```diff
@@ -64,6 +79,13 @@ def is_split_memory(self):
     return 'SPLIT_MEMORY=' in str(self.emcc_args)
   def is_wasm(self):
     return 'BINARYEN' in str(self.emcc_args) or self.is_wasm_backend()
+  def is_linux(self):
+    # sys.platform is 'linux2' in Python 2.x and 'linux' in Python 3.x
+    return sys.platform.startsWith('linux')
+  def is_mac(self):
+    return sys.platform == 'darwin'
+  def is_windows(self):
+    return sys.platform == 'win32'
```

Comment:
I still think that in principle we don't want to duplicate logic. 
`is_windows()` could be implemented as `return WINDOWS`. Even better would be to change shared.WINDOWS into a function or something but not for this PR.